systori/apps/project/test_api.py

```diff
@@ -28,6 +31,52 @@ def test_exists(self):
         self.assertEqual(response.json()["name"], _("Project not found."))
         self.assertEqual(response.status_code, 206)
 
+    def test_list_notes(self):
+        note1 = NoteSerializer(
+            NoteFactory(
+                project=self.project, content_object=self.project, worker=self.worker
+            )
+        ).data
+        note2 = NoteSerializer(NoteFactory(
+            project=self.project, content_object=self.project, worker=self.worker
+        )).data
+        response = self.client.get(f"/api/project/{self.project.pk}/notes/")
+        self.assertEqual(response.status_code, HTTP_200_OK)
+        json = response.json()
+        self.assertDictContainsSubset(
+            json[0],
+            {
+                "text": note1["text"],
+                "created": note1["created"],
+                "project": note1["project"],
+                "worker": note1["worker"],
+                "pk": note1["pk"],
+                "html": note1["html"],
+            },
+        )
+        self.assertDictContainsSubset(
+            json[1],
+            {
+                "text": note2["text"],
+                "created": note2["created"],
+                "project": note2["project"],
+                "worker": note2["worker"],
+                "pk": note2["pk"],
+                "html": note2["html"],
+            },
+        )
+
+    def test_create_note(self):
+        response = self.client.post(
+            f"/api/project/{self.project.pk}/notes/", {"text": "This is a test note"}
+        )
+
+        self.assertEqual(response.status_code, HTTP_201_CREATED)
+        json = response.json()
+        self.assertEqual(json["project"], self.project.pk)
+        self.assertEqual(json["worker"], self.worker.pk)
+        self.assertEqual(json["text"], "This is a test note")
+
```

Comment:
add `test_update_note` for greater coverage?

Reply:
Okay, we don't have the update endpoint yet. Will add that